cursorless-talon/src/spoken_forms.json

```diff
@@ -0,0 +1,260 @@
+{
+  "actions": {
+    "simple_action": {
```

Comment:
Huh. A bit odd that Typescript needs to know about the list names, in snake case, that we expect to use Talon side 🤔

Reply:
I guess maybe this is a reasonable first step, and we can revisit when we actually gen from typescript?

Reply:
No just iterate keys and ignore the value. Python controls the format and ts just updates what's already there was my idea.

Reply:
Huh interesting idea. We can discuss in follow-up to this PR